packages/venia-concept/src/actions/checkout/actions.js

```diff
@@ -0,0 +1,27 @@
+import { createActions } from 'redux-actions';
+
+const prefix = 'CHECKOUT';
+const actionTypes = ['EDIT', 'RESET'];
+
+// classify action creators by domain
+// e.g., `actions.order.submit` => CHECKOUT/ORDER/SUBMIT
+// a `null` value corresponds to the default creator function
+const actionMap = {
+    CART: {
+        SUBMIT: null,
+        ACCEPT: null,
+        REJECT: null
+    },
+    INPUT: {
+        SUBMIT: null,
+        ACCEPT: null,
+        REJECT: null
+    },
+    ORDER: {
+        SUBMIT: null,
+        ACCEPT: null,
+        REJECT: null
+    }
+};
```

Comment:
I got confused by this action map and how it corresponded to our process. What's the difference between submitting a cart and submitting an order? Should the cart part of this belong to cart actions? I think you should put comments in this file, either explaining the purposes of these actions, or linking to another file which makes it more clear.